synapse/config/server.py

```diff
@@ -155,9 +155,10 @@ def default_config(self, server_name, **kwargs):
             # The port to listen for HTTPS requests on.
             port: %(bind_port)s
 
-            # Local interface to listen on.
-            # The empty string will cause synapse to listen on all interfaces.
-            bind_address: ''
+            # Local addresses to listen on.
+            # This will listen on all IPv4 addresses by default.
```

Comment:
I was thinking about that, but also need to specify that having both uncommented at the same time won't work on Linux, so I was thinking about whether to document this in the default config or somewhere else?